index.html

```diff
@@ -3,6 +3,14 @@
 header: Track the money in Oakland elections
 ---
 
+{% assign oakland_total_funds = 0 %}
+{% for source in site.data.totals.oakland-june-2018 %}
+  {% assign oakland_total_funds = oakland_total_funds | plus: source[1] %}
+{% endfor %}
```

Comment:
@sfdoran should the total here be specific to the November election, or should it include both June and November elections?